### PR TITLE
update eth india registration flow

### PR DESCRIPTION
**Pull request by AndrewCLu**
Updated registration for ETH India:
- [x] update cover image
- [x] revert user info screen to regular form
- [x] only allow password registration
- [x] copy for account creation slides

### CODE REVIEW

File: apps/frontend/src/pages/register.tsx [hunk 1]
```diff
@@ -48,8 +48,8 @@ const Register: React.FC = () => {
     return null;
   }
 
-  if (savedTap.tapResponse.chipIssuer === ChipIssuer.DEVCON_2024) {
-    return <RegisterDevcon savedTap={savedTap} />;
```
Comment (stevenelleman) [inline apps/frontend/src/pages/register.tsx:52]: Do we need this for testing purposes? I have some Devcon stickers and wound up using them yesterday to create a test account for Spotify OAuth App request. I may need to create other test accounts for other OAuth App requests.

**Comment by stevenelleman** [comment]
Mind updating the seed file for the new chip issuer?

File: apps/frontend/src/pages/register.tsx [hunk 2]
```diff
@@ -48,8 +48,8 @@ const Register: React.FC = () => {
...
+  if (savedTap.tapResponse.chipIssuer === ChipIssuer.TESTING) {
```
Comment (stevenelleman) [inline apps/frontend/src/pages/register.tsx:51]: Update to correct chip issuer?

File: apps/frontend/src/pages/register.tsx (hunk 1 above)
Comment (RiverRuby) [reply apps/frontend/src/pages/register.tsx:52]: Yes, we should keep this around so we can give people the Devcon stickers in the meantime and have them test the flow

File: apps/frontend/src/pages/register.tsx (hunk 2 above)
Comment (RiverRuby) [reply apps/frontend/src/pages/register.tsx:51]: Yep

File: apps/frontend/src/constants.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export const communitiesHumanReadable: { [key: string]: string } = {
   testing: "Testing",
 };
 
-export const ERROR_SUPPORT_CONTACT = "https://t.me/stevenelleman";
+export const ERROR_SUPPORT_CONTACT = "https://t.me/vivboop";
```
Comment (stevenelleman) [inline apps/frontend/src/constants.ts:15]: I'd be happy to keep helping for Devcon. Maybe make a new constant for EthIndia?

Comment (RiverRuby) [reply apps/frontend/src/constants.ts:15]: all good, i'll handle inbound moving forward!

**Review by stevenelleman — APPROVED** [review]

**Pull request merged**